apps/src/templates/AutocompleteDropdown.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react';
+import Select from 'react-select';
+
+const AutocompleteDropdown = React.createClass({
```

Comment:
I think we're favoring `extends Component` over `creatClass` on new components these days